about_solid.html

```diff
@@ -54,8 +54,8 @@ <h1>Solid</h1>
                         <p class="pt-5">Solid is a specification that lets people store their data securely in
                             decentralized data stores
                             called Pods. Pods are like secure personal web servers for your data.</p>
-                        <p>You control access to the data in your Pod. You decide what data to share and with whom (be
-                            itindividuals, organizations, and/or applications). You can revoke access at any time.</p>
```

Comment:
All these changes from second to third person were made in https://inrupt.invisionapp.com/console/share/FW5K83EZGRJ/987660909/play after Rajnish got to work so just updating that here too.